react-native/react/actions/tracker.js

```diff
@@ -239,8 +240,25 @@ function updateUserInfo (userCard: UserCard, username: string, getState: () => {
 function serverCallMap (dispatch: Dispatch, getState: Function): CallMap {
   /* eslint-disable arrow-parens */
   const sessionIDToUsername: { [key: number]: string } = {}
-  const identifyUi = {
-    start: (params: {sessionID: number, username: string, reason: {reason: string}}) => {
+  type mapType = {
+    start: (param: identifyUi_start_param) => any,
+    displayKey: (param: identifyUi_displayKey_param) => any,
+    reportLastTrack: (param: identifyUi_reportLastTrack_param) => any,
+    launchNetworkChecks: (param: identifyUi_launchNetworkChecks_param) => any,
+    displayTrackStatement: (param: identifyUi_displayTrackStatement_param) => any,
+    finishWebProofCheck: (param: identifyUi_finishWebProofCheck_param) => any,
+    finishSocialProofCheck: (param: identifyUi_finishSocialProofCheck_param) => any,
+    displayCryptocurrency: (param: identifyUi_displayCryptocurrency_param) => any,
+    displayUserCard: (param: identifyUi_displayUserCard_param) => any,
+    reportTrackToken: (param: identifyUi_reportTrackToken_param) => any,
+    confirm: (param: identifyUi_confirm_param) => any,
+    finish: (param: identifyUi_finish_param) => any
+  }
+
+  // TODO the promisifyResponses is actually dangerous. you can't send errors back and it makes typing it more difficult
```

Comment:
the above is all funky cause we're 'generating' the real callmap. This is a little problematic as its turning the return value into the call to the response object. I think we might not want to do this. its not a big deal to pass that in. We can have this return a promise but we don't need this piece